package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "n2words",
-  "version": "1.13.0",
+  "version": "1.14.0",
```

Comment:
You will need to update the `package-lock.json` (via `npm install`) and `dist/n2words.js` too